### DISCUSSION TITLE
请求增加白名单功能

### DISCUSSION THREAD

**Discussion by Lisuaki** [Planshit/Tai#209] [Ideas]
非常喜欢作者的软件！统计使用时间真的超级有帮助的！

不过，因为有很多很多后台只会偶尔弹出的应用或者不需要统计的软件，强迫症每次都要手动一个一个的点忽略和清空统计超麻烦的，如果可以的话希望能增加一个白名单的功能，只记录名单上的应用就好！

十分感谢作者！真的是个很棒的软件b（￣▽￣）d

**Comment by noberumotto** [comment]
在1.5.0.3版本加入了
